CalculatorTest.py

```diff
@@ -20,6 +20,9 @@ def test_run_operation_divide(self):
         with self.assertRaises(ZeroDivisionError):
             Calculator.run_operation(9, 0, '/')
```

Comment:
This is a great edge case to think consider! Maybe also try dividing 0 by 0 to make sure it works as expected with the same number

Reply:
I agree, we could also test division by zero with decimals. For example, 0.0/0.0.